core/dispatcher/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -260,9 +254,9 @@ class Invoker(
                     // completed only once, there is only one completion message sent to the feed as a result.
                     activationFeed ! releaseResource
                     // Since there is no active action taken for completion from the invoker, writing activation record is it.
-                    info(this, "recording the activation result to the data store", INVOKER_RECORD_ACTIVATION_START)
+                    info(this, "recording the activation result to the data store")
                     val result = WhiskActivation.put(activationStore, activation)
-                    info(this, "finished recording the activation result", INVOKER_RECORD_ACTIVATION_DONE)
+                    info(this, "finished recording the activation result")
```

Comment:
These two log statements are redundant (the CouchDB client will log anyway).

The finish log is/was wrong as `WhiskActivation.put` returns a Future which is not waited upon.